docset/winserver2022-ps/defender/Get-MpPerformanceReport.md

```diff
@@ -0,0 +1,331 @@
+---
+description: Use this topic to analyze and report on Microsoft Defender Antivirus (MDAV) performance recordings.
+external help file: MSFT_MpThreat.cdxml-help.xml
+Module Name: Defender
+ms.date: 03/20/2022
+online version: 
+schema: 2.0.0
+title: Get-MpPerformanceReport
+
+---
+
+# Get-MpPerformanceReport
+
+The following section describes the Get-MpPerformanceReport PowerShell cmdlet. Analyzes and reports on Microsoft Defender Antivirus (MDAV) performance recording.
+
+## Syntax: Get-MpPerformanceReport
```

Comment:
You cannot change the H2 and H3 headers created by PlatyPS. This violates the schema and breaks the build.